runtime/binding-tcp/src/main/resources/META-INF/services/jakarta.json.spi.JsonProvider

```diff
@@ -0,0 +1 @@
+org.leadpony.joy.yaml.YamlProvider
```

Comment:
Replicating this here doesn't seem correct, it should be coming from the engine JAR when shading instead.